index-api/src/Ergvein/Index/API/Models.hs

```diff
@@ -0,0 +1,15 @@
+module Ergvein.Index.API.Models where
+
+import Data.Word
+import Data.Text
+import Ergvein.Aeson
+
+type LockingScriptHash = Text
+
+data Currency = BTC | ERGO
+
+data BalanceResponce = BalanceResponce 
+    { confirmed   :: !Word64
+    , unconfirmed :: !Word64
```

Comment:
Add prefix balanceResponceUnconfirmed (perhaps balRespUnconfirmed is better) and for other fields too.

Or rename it to `_unconfirmed` and generate lens field. Ask me for TH lens module.